sound/soc/sof/core.c

```diff
@@ -92,47 +92,9 @@ void snd_sof_get_status(struct snd_sof_dev *sdev, u32 panic_code,
 }
```

Comment:
commit message: "of Device Tree" -> OF/Device Tree?

this restriction

ARM-> OF/Device Tree

Reply:
According to this: https://www.kernel.org/doc/Documentation/devicetree/usage-model.txt
the official name is "Open Firmware Device Tree", or simply Device Tree (DT)"
so it doesn't make sense to say OF/Device Tree (with the slash)

I think the only things needs to change like this:

"In the case, of Device Tree we.." -> "In the case of Device Tree we..." (remove comma).
"have this restrictions" -> "have this restriction"
"allowing for arm platforms" -> "allowing for Device Tree platforms"

Reply:
@ranj063 other than that the change looks good to me!